lms/envs/common.py

```diff
@@ -2146,8 +2149,8 @@
     # API access administration
     'openedx.core.djangoapps.api_admin',
 
-    # Verified Track Content Cohorting
-    'verified_track_content',
+    # Verified Track Content Cohorting (Beta feature that will hopefully be removed)
```

Comment:
No, there isn't one yet. I think creating it is a bit premature, as we have to see how the migration of courses (at rerun time) ends up going.